common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2235,6 +2235,10 @@ public static enum ConfVars {
     HIVE_ICEBERG_ALLOW_DATAFILES_IN_TABLE_LOCATION_ONLY("hive.iceberg.allow.datafiles.in.table.location.only", false,
         "If this is set to true, then all the data files being read should be withing the table location"),
 
+    HIVE_ICEBERG_CATALOG_ACTOR_CLASS("hive.iceberg.catalog.actor.class", "org.apache.iceberg.rest.HMSCatalogActor",
+        "Catalog actor implementation class. Default value is an embedded (no-thrift) catalog actor." +
+            "For a thrift-based single-tenant actor, use \"org.apache.iceberg.hive.HiveCatalogActor\".."),
+
```

Comment:
EmbeddedCatalogActor vs ThriftCatalogActor ?

Reply:
I could not easily distinguish between  EmbeddedCatalogActor and ThriftCatalogActor.
More meaningful names are the best choise. Since this is an Iceberg extension IcebergRestCatalogActor and HiveTriftCatalogActor sound good to me.